providers/mysql/mysql.go

```diff
@@ -37,26 +38,34 @@ var (
 )
 
 type config struct {
-	MySQLURL          string        `file:"url" env:"MYSQL_URL"`
-	MySQLHost         string        `file:"host" env:"MYSQL_HOST" default:"localhost"`
-	MySQLPort         string        `file:"port" env:"MYSQL_PORT" default:"3306"`
-	MySQLUsername     string        `file:"username" env:"MYSQL_USERNAME" default:"root"`
-	MySQLPassword     string        `file:"password" env:"MYSQL_PASSWORD" default:""`
-	MySQLDatabase     string        `file:"database" env:"MYSQL_DATABASE"`
-	MySQLMaxIdleConns uint64        `file:"max_idle_conns" env:"MYSQL_MAXIDLECONNS" default:"1"`
-	MySQLMaxOpenConns uint64        `file:"max_open_conns" env:"MYSQL_MAXOPENCONNS" default:"2"`
-	MySQLMaxLifeTime  time.Duration `file:"max_lifetime" env:"MYSQL_MAXLIFETIME" default:"30m"`
-	MySQLDebug        bool          `file:"debug" env:"MYSQL_DEBUG" default:"false"`
-	MySQLCharset      string        `file:"charset" env:"MYSQL_CHARSET" default:"utf8mb4"`
-	MySQLTLS          string        `file:"tls" env:"MYSQL_TLS" default:"false"`
+	MySQLURL            string        `file:"url" env:"MYSQL_URL"`
+	MySQLHost           string        `file:"host" env:"MYSQL_HOST" default:"localhost"`
+	MySQLPort           string        `file:"port" env:"MYSQL_PORT" default:"3306"`
+	MySQLUsername       string        `file:"username" env:"MYSQL_USERNAME" default:"root"`
+	MySQLPassword       string        `file:"password" env:"MYSQL_PASSWORD" default:""`
+	MySQLDatabase       string        `file:"database" env:"MYSQL_DATABASE"`
+	MySQLMaxIdleConns   uint64        `file:"max_idle_conns" env:"MYSQL_MAXIDLECONNS" default:"1"`
+	MySQLMaxOpenConns   uint64        `file:"max_open_conns" env:"MYSQL_MAXOPENCONNS" default:"2"`
+	MySQLMaxLifeTime    time.Duration `file:"max_lifetime" env:"MYSQL_MAXLIFETIME" default:"30m"`
+	MySQLDebug          bool          `file:"debug" env:"MYSQL_DEBUG" default:"false"`
+	MySQLCharset        string        `file:"charset" env:"MYSQL_CHARSET" default:"utf8mb4"`
+	MySQLTLS            string        `file:"tls" env:"MYSQL_TLS"`
+	MySQLCaCertAddr     string        `file:"mysql_ca_cert_addr" env:"MYSQL_CA_CERT_ADDR"`
```

Comment:
In my opinion, here `PATH` is better than `ADDR`, how do you think? @sfwn

Reply:
agree